### ISSUE TITLE
FAMD fit method not compatible with scikit-learn pipeline

### ISSUE THREAD

**Issue by ustcwxz** [MaxHalford/prince#165]
```
X_train, X_test, y_train, y_test = train_test_split(X, y)
pip = Pipeline(steps=[('transformer', prince.FAMD(n_components=13)), ('estimator', DecisionTreeRegressor())])
pip.fit(X=X_train, y=y_train)
```
X is a pandas DataFrame
y is a pandas Series
The fit method throw error "ValueError: The truth value of a Series is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all()."

That because the pipeline's fit method need params "res = transformer.fit_transform(X, y, **fit_params)"

**Comment by MaxHalford** [comment]
Fixed in version 0.12.1 which has just been released. Let me know if something is wrong :)

**Comment by ustcwxz** [comment]
Thanks! It's working great now!
